src/core/i18n/translation/locales/en.ts

```diff
@@ -95,7 +95,7 @@ export const translation: ITranslations = {
                 retry: 'Retry',
                 reset: 'Reset',
                 resetAll: 'Reset all',
-                restake: 'Restake',
+                restake: 'Switch Node',
```

Comment:
We should not do this, if you need `App.labels.restake` in another context, you'll get `Switch Node` instead

Reply:
We only use App.labels.restake in one instance: zilliqa config. But I can modify the name if you want

Reply:
Yes, we should keep the key value pair from en.ts self explanatory.